### PR TITLE
added Persian locale

### PR DESCRIPTION
**Pull request by ali-sokhandan**

### CODE REVIEW

File: locale/fa_IR/emailTemplates.xml [hunk 1]
```diff
@@ -0,0 +1,1105 @@
...
+		<description>This email is sent to registered readers via the "Notify Users" link in the Editor's User Home. It notifies readers of a new issue and invites them to visit the journal at a supplied URL.</description>
+	</email_text>
+	<email_text key="LOCKSS_NEW_ARCHIVE">
+		<subject>درخواست بایگانی کردن {$contextName}:</subject>
+		<body><![CDATA[مسئول محترم کتابخانه دانشگاه:<br />
+{$contextName} &amp;lt;<a href="{$contextUrl}">{$contextUrl}</a>&amp;gt; ، عضو دانشکده شما می باشد &amp;lt;[name of member]&amp;gt;. این مجله خواستار ایجاد سیستم بایگانی LOCKSS با آن کتابخانه و کتابخانه های دیگر دانشگاهها می باشد.<br />
+<br />
+[Brief description of journal]<br />
+<br />
+برنامه LOCKSS تحت آدرس http://lockss.org که یک ابتکار بین المللی مربوط به ناشر/کتابخانه می باشد. این نرم افزار رایگان، حتی توسط یک رایانه شخصی نیز اجرا می گردد. کل سامانه آن لاین شده است و به نگهداری مختصری نیازمند است.<br />
```
Comment (asmecher) [inline locale/fa_IR/emailTemplates.xml]: It looks like this is incompletely translated -- is that intentional?

File: locale/fa_IR/editor.xml [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    <message key="editor.submissions.assignedTo">منتسب شده به</message>
+    <message key="editor.submissions.inSection">برای مقالات از نوع</message>
+    <message key="editor.article.schedulePublication">برنامه چاپ</message>
+    <message key="editor.article.scheduleForPublication">کاندید پذیرش</message>
+    <message key="editor.article.scheduleForPublication.toBeAssigned">قرار است انتساب یابد</message>
+    <message key="editor.article.payment.publicationFeeNotPaid">هزینه چاپ پرداخت نشده است. برای قرار دادن در نوبت چاپ به نویسنده اطلاع دهید تا هزینه را پرداخت کند</message>
+    <message key="editor.article.display">نمایش</message>
+    <message key="editor.article.hideTocAuthorDescription">حذف نام نویسندگان از فهرست مقالات شماره مجله</message>
+    <message key="editor.article.hideTocAuthorDefault">پیش‌فرض</message>
+    <message key="editor.article.hideTocAuthorHide">حذف</message>
```
Comment (asmecher) [inline locale/fa_IR/editor.xml]: Use tab-based indents rather than spaces if possible.

File: locale/fa_IR/emailTemplates.xml [hunk 3]
```diff
@@ -0,0 +1,1105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE email_texts SYSTEM "../../lib/pkp/dtd/emailTemplateData.dtd">
+
+<!--
+  * locale/fa_IR/emailTemplates.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
```
Comment (asmecher) [inline locale/fa_IR/emailTemplates.xml]: Copyrights should be to Simon Fraser University, not Simon Fraser University Library.

File: locale/fa_IR/emailTemplates.xml [hunk 4]
```diff
@@ -0,0 +1,1105 @@
...
+  * Copyright (c) 2003-2016 John Willinsky
```
Comment (asmecher) [inline locale/fa_IR/emailTemplates.xml]: Copyright dates to John Willinsky or Simon Fraser University should be -2018.

File: locale/fa_IR/emailTemplates.xml (hunk 1 above)
Comment (ali-sokhandan) [reply locale/fa_IR/emailTemplates.xml]: all section check and the untranslated ones has been translated. only the descriptions translation is incomplete

**Comment by ali-sokhandan** [comment]
dear @asmecher the mentioned changes has been applied.

File: plugins/blocks/subscription/locale/fa_IR/locale.xml [hunk 5]
```diff
@@ -1,25 +1,23 @@
...
 <!--
   * plugins/blocks/subscription/locale/fa_IR/locale.xml
   *
-  * Copyright (c) 2013-2017 Simon Fraser University
-  * Copyright (c) 2003-2017 John Willinsky
+  * Copyright (c) 2013-2018 Simon Fraser University
+  * Copyright (c) 2003-2018 John Willinsky
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
   *
-  * Credits: https://pkp.sfu.ca/wiki/index.php?title=OJS:_Farsi_(fa_IR)
```
Comment (asmecher) [inline plugins/blocks/subscription/locale/fa_IR/locale.xml]: Would you be willing to leave a notice like this in the headers, and then we could add credit in the wiki? We're trying to keep our code headers from accumulating too many names -- and we want to be fair when translations are adopted or work is shared.

**Comment by asmecher** [comment]
(Noted at https://github.com/pkp/pkp-lib/issues/3320)

**Comment by ali-sokhandan** [comment]
@asmecher 
the name comment replaced with credits on

**Comment by asmecher** [comment]
Excellent, thanks, @ali-sokhandan! How would you/your institution like to be credited on the wiki?

**Comment by ali-sokhandan** [comment]
at this time i am nobody from nowhere. if its possible leave it empty or any thing you prefer.

**Comment by asmecher** [comment]
I do prefer to give credit for work (beyond what's already visible on github, of course) but it's up to you. If you don't mind having your name added to the wiki, let me know how you'd like it written and I'll add it! It doesn't matter whether you have an institution.

**Comment by ali-sokhandan** [comment]
please simply only add my name: Alireza Sokhandan, and if contact info required: ali.sokhandan@gmail.com

thanks so much. :)

**Comment by asmecher** [comment]
Thanks for contributing!

**Pull request merged**